server/documents/elements/icon.html.eco

```diff
@@ -144,44 +170,44 @@ themes      : ['Default']
         <div class="column"><i class="arrows alternate icon" data-search-terms="arrow, arrows, bigger, enlarge, expand, fullscreen, move, position, reorder, resize"></i>arrows alternate</div>
-        <div class="column"><i class="arrows alternate horizontal icon" data-search-terms="arrows-h, resize"></i>arrows alternate horizontal</div>
-        <div class="column"><i class="arrows alternate vertical icon" data-search-terms="arrows-v, resize"></i>arrows alternate vertical</div>
-        <div class="column"><i class="caret down icon" data-search-terms="arrow, dropdown, menu, more, triangle down"></i>caret down</div>
-        <div class="column"><i class="caret left icon" data-search-terms="arrow, back, previous, triangle left"></i>caret left</div>
-        <div class="column"><i class="caret right icon" data-search-terms="arrow, forward, next, triangle right"></i>caret right</div>
-        <div class="column"><i class="caret square down icon" data-search-terms="caret-square-o-down, dropdown, menu, more"></i>caret square down</div>
-        <div class="column"><i class="caret square down outline icon" data-search-terms="caret-square-o-down, dropdown, menu, more"></i>caret square down outline</div>
-        <div class="column"><i class="caret square left icon" data-search-terms="back, caret-square-o-left, previous"></i>caret square left</div>
-        <div class="column"><i class="caret square left outline icon" data-search-terms="back, caret-square-o-left, previous"></i>caret square left outline</div>
-        <div class="column"><i class="caret square right icon" data-search-terms="caret-square-o-right, forward, next"></i>caret square right</div>
-        <div class="column"><i class="caret square right outline icon" data-search-terms="caret-square-o-right, forward, next"></i>caret square right outline</div>
-        <div class="column"><i class="caret square up icon" data-search-terms="caret-square-o-up"></i>caret square up</div>
-        <div class="column"><i class="caret square up outline icon" data-search-terms="caret-square-o-up"></i>caret square up outline</div>
-        <div class="column"><i class="caret up icon" data-search-terms="arrow, triangle up"></i>caret up</div>
-        <div class="column"><i class="cart arrow down icon" data-search-terms="shopping"></i>cart arrow down</div>
-        <div class="column"><i class="chart line icon" data-search-terms="activity, analytics, dashboard, gain, graph, increase, line-chart"></i>chart line</div>
-        <div class="column"><i class="chevron circle down icon" data-search-terms="arrow, dropdown, menu, more"></i>chevron circle down</div>
+        <div class="column"><i class="arrows alternate horizontal icon" data-search-terms="arrows-h, expand, horizontal, landscape, resize, wide"></i>arrows alternate horizontal</div>
+        <div class="column"><i class="arrows alternate v icon" data-search-terms="arrows-v, expand, portrait, resize, tall, vertical"></i>arrows alternate v</div>
```

Comment:
This abbreviation `v` conflict the description in the header.
Should be `vertical`?

```
abbreviation are replaced by full words
i.icon.*-h         =>  i.icon.*.horizontal
i.icon.*-v         =>  i.icon.*.vertical
```

Reply:
https://github.com/fomantic/Fomantic-UI/pull/730#discussion_r282263489